packages/darklang/languageTools/programTypes.dark

```diff
@@ -1,60 +1,54 @@
 module Darklang =
   module LanguageTools =
     module ProgramTypes =
-      /// Used to name where type/function/etc lives, eg a BuiltIn module, a User module,
-      /// or a Package module.
-      module FQName =
 
-        /// A name that is built into the runtime
-        type BuiltIn<'name> =
-          { modules: List<String>
-            name: 'name
-            version: Int64 }
+      module FQTypeName =
 
-        /// Part of the user's program (eg canvas or cli)
-        type UserProgram<'name> =
-          { modules: List<String>
-            name: 'name
-            version: Int64 }
+        type UserProgram = { modules: List<String>; name: String; version: Int64 }
```

Comment:
TODO: add comment back
/// Part of the user's program (eg canvas or cli)
type UserProgram